src/Html.php

```diff
@@ -4923,6 +4924,8 @@ public static function jsAjaxDropdown($name, $field_id, $url, $params = [])
             'display_emptychoice' => false,
             'specific_tags'       => [],
             'parent_id_field'     => null,
+            'templateResult'      => null,
+            'templateSelection'   => null,
```

Comment:
To prevent any BC break, the values here should be identicat to the values that were previously hardcoded.

```suggestion
            'templateResult'      => 'templateResult',
            'templateSelection'   => 'templateSelection',
```